cmd/alfalfa/make.go

```diff
@@ -45,18 +46,22 @@ func runMake(c *cli.Context) error {
 			log.Fatal(ctx, err)
 		}
 		res, _ := json.MarshalIndent(s, "", "  ")
-		if err := ioutil.WriteFile("./alexa/skill.json", res, 0644); err != nil {
+		if err := ioutil.WriteFile("./alexa/skill.json", res, 0o644); err != nil { //nolint:gosec
```

Comment:
to get rid of the gosec you need to wrap the path in `filepath.Clean` if I remember correctly.

Reply:
that didn't solve it. I needed to disable this check in the linter.